packages/suite/src/actions/wallet/signVerifyActions.ts

```diff
@@ -58,50 +84,72 @@ export const verify = (address: string, message: string, signature: string, hex
     dispatch: Dispatch,
     getState: GetState,
 ) => {
-    const selectedDevice = getState().suite.device;
-    if (!selectedDevice) return;
-    const error = validateAddress(address);
+    const { device } = getState().suite;
+    const { account } = getState().wallet.selectedAccount;
+    if (!device || !account) return;
+
+    let fn;
+    const params = {
+        address,
+        message,
+        signature,
+        coin: account.symbol,
+        hex,
+    };
+    const error = isAddressValid(address, params.coin as Account$['symbol'])
```

Comment:
use `account.symbol` instead of `params.coin as Account$['symbol']`

Reply:
I couldn't have known this, this is the first serious component in my work on this project, i rewrote this